motionplan/motionPlanner.go

```diff
@@ -177,19 +177,21 @@ func Replan(ctx context.Context, request *PlanRequest, currentPlan Plan, replanC
 		return nil, err
 	}
 
-	// TODO: RSDK-3236 this should seed off of currentPlan
 	resultSlices, err := sfPlanner.PlanSingleWaypoint(
 		ctx,
 		request.StartConfiguration,
 		request.Goal.Pose(),
 		request.WorldState,
 		request.ConstraintSpecs,
+		currentPlan,
 		request.Options,
 	)
 	if err != nil {
 		return nil, err
 	}
 	newPlan := sf.inputsToPlan(resultSlices)
+	request.Logger.Debugf("seedplan %v", currentPlan)
+	request.Logger.Debugf("plan %v", newPlan)
```

Comment:
isn't the final plan we return being logged elsewhere? Concerned about making things noisy

Reply:
Ah; these can be removed. I missed that.